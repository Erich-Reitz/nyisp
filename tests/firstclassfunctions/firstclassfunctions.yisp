(define twice (f x)
  (f (f x))
)

(define square (x) 
  (* x x)
)

(set sq square)

(twice sq 2) ; ==> 16