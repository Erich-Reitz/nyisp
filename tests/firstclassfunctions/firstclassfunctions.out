twice
square
nil
16
